src/lib/util/util.ts

```
@@ -674,126 +800,19 @@
-                        getLocaleString(
-                            guildLocaleCode,
-                            'CMD.SETTINGS.MISC.FORMAT.OPTIONS.DETAILED',
-                        ),
-                    )
-                    .setStyle(
-                        current === 'detailed'
-                            ? ButtonStyle.Success
-                            : ButtonStyle.Secondary,
-                    )
-                    .setDisabled(current === 'detailed'),
+            const { components, containers: container } = buildSettingsPageFormatOptions(
```

ESLint

[error] 'components' is already declared in the upper scope on line 779 column 21.

(@typescript-eslint/no-shadow)